.vscode/launch.json

```diff
@@ -26,6 +26,7 @@
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen",
```

Comment:
generally, I would say put this to .gitignore :D

Reply:
Yeah, I normally keep .vscode out of the codebase, but it is quite useful to distribute a setup which shows how to run tests and setup a local instance with breakpoints with probot, since it is not straightforward to do :)